go.mod

```diff
@@ -54,6 +52,8 @@ require (
 	gopkg.in/yaml.v3 v3.0.1
 )
 
+require github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/appcontainers/armappcontainers/v2 v2.0.0-beta.3
```

Comment:
Not really sure why `go mod tidy` put this reference outside of the above require block.